augtool.c:450:30: warning: format specifies type 'long' but the argument has type 'time_t' (aka 'long long') [-Wformat

Index: src/augtool.c
--- src/augtool.c.orig
+++ src/augtool.c
@@ -454,7 +454,7 @@ static void print_time_taken(const struct timeval *sta
                              const struct timeval *stop) {
     time_t elapsed = (stop->tv_sec - start->tv_sec)*1000
                    + (stop->tv_usec - start->tv_usec)/1000;
-    printf("Time: %ld ms\n", elapsed);
+    printf("Time: %lld ms\n", elapsed);
 }
 
 static int run_command(const char *line, bool with_timing) {
